recipes-aws/python/python3-awscrt_0.10.8.bb

```diff
@@ -25,6 +25,10 @@ SRC_URI[sha256sum] = "c6859e35c57922f175b587e02bfb662ce30a06c9aa688a7303112c91e2
 do_configure_prepend() {
     sed -i \
         -e 's:AWS_LIBCRYPTO_INSTALL = None:AWS_LIBCRYPTO_INSTALL = "${RECIPE_SYSROOT}/usr":' \
-        -e 's:/lib/:${base_libdir}/:' \
         ${S}/setup.py
+    if ${@bb.utils.contains('DISTRO_FEATURES','usrmerge','true','false',d)}; then
+        echo  # do nothing
+    else
```

Comment:
Can the if statement be changed to use the "!" operator or reverse "true" and "false" in the contains statement to simplify this. Admittedly from a readability perspective this works but the "echo" fall-through I assume can be avoided to tidy this up.